### PR TITLE
Peacock

### PR DESCRIPTION
**Pull request by UnknownHero99**

### CODE REVIEW

**Comment by UnknownHero99** [comment]
Now all of items get ID, changed API URL

File: streamingscrapper/spiders/peacock.py [hunk 1]
```diff
@@ -229,7 +231,7 @@ def parse_program_linear(self, program, loader):
         end_time = datetime.datetime.fromtimestamp(program["startTimeEpoch"]) + datetime.timedelta(
             seconds=program["durationInSeconds"])
         loader.add_value("end_time", "{0:%Y-%m-%dT%H:%M:%S}".format(end_time))
-        loader.add_value("program_url", linear_visit_url + loader.get_value("channelId"))
```
Comment (kodmarcel) [inline streamingscrapper/spiders/peacock.py]: why is this removed?

File: streamingscrapper/spiders/peacock.py [hunk 2]
```diff
@@ -194,10 +191,11 @@ def parse_program(self, program, response):
 
         return loader.load_item()
 
-    def parse_program_linear(self, program, loader):
+    def parse_program_linear(self, program, channel_id, channel_name):
```
Comment (kodmarcel) [inline streamingscrapper/spiders/peacock.py:194]: where do you add those?

Comment (UnknownHero99) [reply streamingscrapper/spiders/peacock.py:194]: Those are added when function is being called in line 112
`yield self.parse_program_linear(schedule, channel_id, channel_name)`

File: streamingscrapper/spiders/peacock.py (hunk 1 above)
Comment (UnknownHero99) [reply streamingscrapper/spiders/peacock.py]: this is removed because i changed how parameters get passed to parse_program_linear as it was not implemented correctly, therefore another line (209) was added where are other fields that require channel_id (to group them)

`loader.add_value("program_url", linear_visit_url + channel_id)`

File: streamingscrapper/spiders/peacock.py (hunk 2 above)
Comment (kodmarcel) [reply streamingscrapper/spiders/peacock.py:194]: somehow i managed to miss it completely - i didnt see the lines 207-210

all fine

File: streamingscrapper/spiders/peacock.py (hunk 1 above)
Comment (kodmarcel) [reply streamingscrapper/spiders/peacock.py]: somehow i managed to miss it completely - i didnt see the lines 207-210

all fine
